<launch>
    <node name="manyears_node"
          pkg="manyears_ros"
          type="manyears_node_standalone"
          launch-prefix="xterm -e gdb --args"
          output="screen">
        <param name="config_file" 
               value="$(find manyears_ros)/data/xtion_stereo.mes"/>
        <rosparam>
            frame_id:    micro_center_link
            microphones:
             - {pos: {x: 0.00, y:  0.05, z: 0.00}, gain: 1.0}
             - {pos: {x: 0.00, y: -0.05, z: 0.00}, gain: 1.0}
             - {pos: {x: 0.00, y: -0.05, z: 0.00}, gain: 0.0}
             - {pos: {x: 0.00, y: -0.05, z: 0.00}, gain: 0.0}
             - {pos: {x: 0.00, y: -0.05, z: 0.00}, gain: 0.0}
             - {pos: {x: 0.00, y: -0.05, z: 0.00}, gain: 0.0}
             - {pos: {x: 0.00, y: -0.05, z: 0.00}, gain: 0.0}
             - {pos: {x: 0.00, y: -0.05, z: 0.00}, gain: 0.0}
        </rosparam>
    </node>

    <node name="sound_pose" pkg="manyears_ros" type="position_exploitation">
        <remap from="/sources_info" to="/tracked_sources"/>
    </node>

    <!-- TEMP -->
    <node name="micro_tf" 
          pkg="tf" 
          type="static_transform_publisher"
          args="0 0 0 0 0 0 1 base_link micro_center_link 10"
    />

</launch>
